bennettbot/job_configs.py

```diff
@@ -331,6 +336,12 @@
                 "action": "schedule_job",
                 "job_type": "show_all",
             },
+            {
+                "command": "show-failed",
+                "help": "Summarise GitHub Actions workflow runs for repos in all three organisations, skipping repos whose runs are all successful.",
```

Comment:
Maybe just "in all organisations" (so we don't need to update this when we add a the new bennett org)